zilliqa/src/cfg.rs

```diff
@@ -535,13 +567,23 @@ pub fn total_native_token_supply_default() -> Amount {
     Amount::from(21_000_000_000_000_000_000_000_000_000)
 }
 
+pub fn genesis_fork_default() -> Fork {
```

Comment:
Shouldn't we copy the old default, with the comment? (where all of these are `true` instead)

>     /// The default implementation of [Forks] returns a single fork at the genesis block, with the most up-to-date
>     /// execution logic.